WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -108,6 +111,8 @@ extension FeatureFlag: OverrideableFlag {
             return "Gutenberg Modal Layout Picker"
         case .whatIsNew:
             return "What's New / Feature Announcement"
+        case .avatarsBasicProfile:
+            return "Show Basic Profile from Avatars"
```

Comment:
It would be better to add a bit more description to specify which avatars will actually show the profile

Reply:
Done, updated the description.